### PR TITLE
Introduce option to enable SSL encryption for LocalTrajectoryUploader

### PR DESCRIPTION
**Pull request by cschuet**

### CODE REVIEW

File: cartographer/cloud/proto/map_builder_server_options.proto [hunk 1]
```diff
@@ -24,4 +24,5 @@ message MapBuilderServerOptions {
   int32 num_event_threads = 3;
   cartographer.mapping.proto.MapBuilderOptions map_builder_options = 4;
   string uplink_server_address = 5;
+  bool enable_ssl_encryption = 7;
```
Comment (gaschler) [inline cartographer/cloud/proto/map_builder_server_options.proto]: just pointing out that 6 would be next

**Comment by cschuet** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user cschuet. Merge queue now has a length of 3.

**Comment by wally-the-cartographer** [comment]
Refusing to merge, since the combined checks state is failure

**Comment by cschuet** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user cschuet. Merge queue now has a length of 1.

**Pull request merged**